recipes/configs/llama3_2_vision/11B_full.yaml

```diff
@@ -0,0 +1,78 @@
+# Config for single device full finetuning in full_finetune_single_device.py
```

Comment:
super nit: `full_finetune_distributed` instead of `full_finetune_single_device`?